ERCS/erc-201.md

```diff
@@ -0,0 +1,147 @@
+---
+eip: 201
```

Comment:
```suggestion
eip: 7849
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.